apps/employee-monitoring/src/components/modal/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx

```diff
@@ -163,10 +146,10 @@ const AddFieldSsModal: FunctionComponent<AddFieldSsModalProps> = ({
   });
 
   // on submit
-  const onSubmit = async (data: any) => {
+  const onSubmit = async () => {
     // extract the unnecessary items for posting
-    const { scheduleName, customGroupName, id, customGroupId, ...rest } =
-      currentScheduleSheet;
+    const { scheduleName, customGroupName, id, customGroupId, ...rest } = currentScheduleSheet;
+    console.log(rest);
```

Comment:
remove later

Reply:
Will remove after the BE fixes the adding of employees to scheduling sheet